test/e2e/logforwarding/fluent/crio_split_log_test.go

```diff
@@ -0,0 +1,71 @@
+package fluent_test
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+	"github.com/openshift/cluster-logging-operator/test/client"
+	"github.com/openshift/cluster-logging-operator/test/helpers/fluentd"
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+	"github.com/openshift/cluster-logging-operator/test/matchers"
+	"github.com/openshift/cluster-logging-operator/test/runtime"
+	"path/filepath"
+	rt "runtime"
+	"strings"
+)
+//E2E test for checking correct reassembly after splitting big logs by CRI-O.
+//LogGenerator container will put into stdout prepared huge Java stacktrace in JSON format.
+//CRI-O will spilt it in to the parts, like:
+// 	2021-04-06T00:17:09.669794202Z stdout P First part of JSON log entry
+//	2021-04-06T00:17:09.669794202Z stdout P Second part of JSON log entry
+//	2021-04-06T00:17:10.113242941Z stderr F Last part of JSON log entry
+//After that in should be assembled by fluent-plugin-concat and stored in original state.
+
+var _ = Describe("[ClusterLogForwarder]", func() {
+	var (
+		c *client.Test
+		f *Fixture
+	)
+
+	BeforeEach(func() { c = client.NewTest() })
+	AfterEach(func() { c.Close() })
+
+	Context("with application receiver", func() {
+		It("Checking reassembly after CRI-O log splitting, for big Java stacktrace ", func() {
+			log := getTrace()
+			containerName := "log-generator-" + strings.ToLower(string(utils.GetRandomWord(7)))
+			f = &Fixture{
```

Comment:
Can you use "NewFixture" and just replace the geneator?  https://github.com/openshift/cluster-logging-operator/blob/48227a1c81a50da7ab6c440a9a84d3569097da74/test/e2e/logforwarding/fluent/fixture_test.go#L21